applications/nrf_desktop/configuration/nrf52kbd_nrf52832/prj.conf

```diff
@@ -28,7 +28,7 @@ CONFIG_CAF_CLICK_DETECTOR=y
 CONFIG_CAF_LEDS=y
 
 CONFIG_CAF_SETTINGS_LOADER_USE_THREAD=y
-CONFIG_CAF_SETTINGS_LOADER_THREAD_STACK_SIZE=1024
+CONFIG_CAF_SETTINGS_LOADER_THREAD_STACK_SIZE=1200
```

Comment:
Is there a reason to set it here explicitly if it is already set by default in `subsys/caf/modules/Kconfig.settings_loader` to the same value?

Reply:
Actually you are right that we can remove the explicit assignment here. Changed.